LayoutTests/imported/w3c/web-platform-tests/fullscreen/api/fullscreen-reordering.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<title>Re-requesting fullscreen doesn't fail but doesn't change order</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<script src="../../html/semantics/popovers/resources/popover-utils.js"></script>
```

Comment:
nit: as a followup, it might be good to make this into an ES module... then it's more explicit where isTopLayer and blessTopLayer are coming from.